inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/ck/ClickHouseConfig.java

```diff
@@ -30,45 +35,61 @@
  * Clickhouse config information, including url, user, etc.
  */
 @Component
+@Service
+@Slf4j
 public class ClickHouseConfig {
 
+    @Autowired
+    private AuditQuerySourceConfigEntityMapper querySourceConfigEntityMapper;
     private static volatile DataSource source;
+    private static volatile String currentJdbcUrl = null;
```

Comment:
Suggest just jdbcUrl

Reply:
"jdbcUrl" has been defined, so I kept this naming.